src/server/server_request_responder.hpp

```diff
@@ -163,9 +164,13 @@ template<typename T_CONN>
 server_request_responder<T_CONN>::server_request_responder ()
   : m_threads_context_manager (std::make_unique<cubthread::system_worker_entry_manager> (TT_SYSTEM_WORKER))
 {
-  const auto THREAD_COUNT = 4; // std::thread::hardware_concurrency ();
-  // TODO configurable and register it to cubthread::manager (set_max_thread_count_from_config)
-  const auto TASK_MAX_COUNT = THREAD_COUNT * 4;
+  const int prm_thread_count = prm_get_integer_value (PRM_ID_SCAL_PERF_PS_REQ_RESPONDER_THREAD_COUNT);
```

Comment:
I've added another request_responder in the page server for the catchup connection_handler, `m_follower_responder`. Regarding this, we should either consider this to the thread number.
 Or it's better to both use one responder, which I tried, but failed.

Reply:
Besides this, I think I should add a work pool to `page_server`, for the catchup thread that I'm working on. This needs the `thread::entry` to append pages through the log manager. By my analysis, CUBRID only have two legal ways to get an entry, either with daemon or workepool. In my opinion, the catchup execution is a one-shot execution, so it's not a work for a daemon. 
All in all, we end up using three worker pools in the page_server. All of these have to be taken into considerations here. 
I doubt that it's a fair choice to introduce that many work pools, though. So, how about using just one work pool and share it with two request responders?
Welcome to any opinion. Actually, I'd like to have your opinion on the implementations I mentioned. Thank you.